translations/pt/resources/model-traits-worksheet-PT.md

```diff
@@ -0,0 +1,21 @@
+# Ficha de Características do Modelo
+
+É provável que um modelo seja descrito por várias características diferentes. Revise a lista abaixo e escolha qual se aplica ao Contexto em que você está trabalhando ou pense em suas próprias características.
```

Comment:
Maybe you can use "contexto" (no capital letter).

Reply:
No problem. The reason why I chose to capitalize was to highlight that word is related a DDD concept, but let's keep it in lowercase, being match with English original version.